generator/generator-opc-generator/generators/app/templates/src/_style.scss

```diff
@@ -0,0 +1,3 @@
+p {
+  font-size: var(--op-component--text-size, 16px);
+}
```

Comment:
@hybridx it'd be really really good if you provide the scss file generated with sections to add the following:

- host styles
- variable definitions

something like this 

```
@import "../../op-styles/op-styles";

$LOCAL: <component-name>;

$variables: ( 
  //Add your variables here
);

:host {
  // add root styles here
}

```

Reply:
Added the styles base code 
```
@import "../../opc-styles/opc-styles.scss";

$LOCAL: <%= componentName %>;

$variables: ( 
  //Add your variables here
);

:host {
  // add root styles here
}

```
After this is merged we would be able to add a package for styles as well